<launch>

    <node name="ros_rqt" pkg="rqt_reconfigure" type="rqt_reconfigure"/>	

    <node name="ros_socket" pkg="ROS_Socket" type="HiwinRA605_socket_ros_test_v1.py" output="screen"/>

    <node name="arm_ros" pkg="ROS_Socket" type="Hiwin_RT605_Arm_Command.py" output="screen"/>

    <node name="ros_strategy" pkg="ROS_Socket" type="HiwinRT605_Strategy_test_v1.py" output="screen"/>

</launch>
